ncm-ntpd/src/main/pan/components/ntpd/schema.pan

```diff
@@ -112,43 +141,73 @@ type ntpd_statistics = {
     "sysstats" ? boolean
 };
 
-type ntpd_filegen_name = string
-    with match(SELF, '^(clock|crypto|loop|peer|raw|sys)stats$');
-
+@documentation{
+    Monitoring/statistics options, see ntp_mon manpage.
+}
 type ntpd_filegen = {
-    "name" : ntpd_filegen_name
+    "name" : string with match(SELF, '^(clock|crypto|loop|peer|raw|sys)stats$')
     "file" : string
     "type" ? string with match(SELF, '^(none|pid|day|week|month|year|age)$')
     "linkornolink" ? string with match(SELF, '^(no)?link$')
     "enableordisable" ? string with match(SELF, '^(en|dis)able$')
 };
 
-type component_ntpd_type = extensible  {
+type ${project.artifactId}_component = extensible  {
```

Comment:
This being extensible looks like it has no effect/value since I don't see anything in the code that suggests that arbitrary additional keys will be added to the config file?

Looking closer I see it is used fairly liberally for the types defined here - perhaps they could/should all be removed?